src/components/CartPage.vue

```diff
@@ -1,43 +1,34 @@
 <template>
   <div class="">
     <!--<h3><q-icon name="shopping_cart"/></h3>-->
-    <p class="text-bold text-tertiary" style="padding: 10px;" v-show="Object.keys(carts).length == 0">
+    <p class="text-bold text-tertiary" style="padding: 10px;" v-show="!hasNonEmptyCart">
       Please add some products to cart :/
     </p>
 
-    <div v-show="Object.keys(carts).length > 0">
+    <div v-show="hasNonEmptyCart">
       <q-list multiline no-border inset-separator	sparse highlight
-              v-show="Object.keys(carts).length > 0"
+              v-show="hasNonEmptyCart"
               v-for="(cart, key) in carts" :key="key">
-        <q-list-header class="text-bold" inset><q-icon name="shopping_cart"/> {{ cart.store_name }}</q-list-header>
-        <q-item class="bg-white" v-for="p in cart.products" :key="p.asset_id" v-if="p.quantity > -1">
-
+        <q-list-header v-if="cart.totals.subtotal" class="text-bold" inset>
+          <q-icon name="shopping_cart"/>{{ formattedTitle(cart.store_name) }}
+        </q-list-header>
+        <q-item class="bg-white" v-for="p in cart.products" :key="p.asset_id" v-if="p.quantity && cart.totals.subtotal">
           <q-item-side :avatar="p.image">
             <q-item-tile>
-              {{ p.quantity }} x {{ (p.price_cents/100).toLocaleString('en-US', {style: 'currency', currency: 'USD',}) }}
+              {{ p.quantity }} x {{ formattedPrice(p.price_cents) }}
             </q-item-tile>
           </q-item-side>
-          <q-item-main v-if="p.title.length > 35">{{ p.title.substring(0,35) }}...</q-item-main>
-          <q-item-main v-if="p.title.length <= 35">{{ p.title }}</q-item-main>
+          <q-item-main>{{ formattedTitle(p.title) }}</q-item-main>
           <q-icon name="delete" color="negative" @click="removeFromCart(p, cart)"></q-icon>
         </q-item>
-        <br>
-        <span class="text-bold" style="padding-left: 20px;">Cart Total:</span>
-          {{ (cart.totals.total/100).toLocaleString('en-US', {
-            style: 'currency',
-            currency: 'USD',
-          }) }}
+        <q-list slot="footer" v-if="cart.totals.subtotal">
+          <br>
+          <span class="text-bold" style="padding-left: 20px;">Cart Subtotal:</span>
+          <span>{{ formattedPrice(cart.totals.subtotal) }}</span>
+          <button :disabled="true" @click="" class="on-right">Checkout</button>
+        </q-list>
       </q-list>
-      <br>
```

Comment:
i may have prematurely done this but i thought it made sense to remove the overall price of all carts right now since you cant check them all out at the same time. the user has to go cart by cart so it made sense to me to leave the cart subtotals and remove the all encompassing total. It also psychs people out when they realize how much their spending since math aint alot o ppls strong suites ;)